docker-compose.yml

```diff
@@ -24,6 +24,7 @@ services:
           - db
         environment:
           - DEV_DATABASE_URL=postgresql://meep:supersafe@db:5432/meep_api
+          - TEST_DATABASE_URL=postgresql://meep:supersafe@db:5432/meep_api
```

Comment:
If you look at  the db block of ```docker-compose.yml```:
```
    db:
        image: postgres:11.3
        environment:
          - POSTGRES_USER=meep
          - POSTGRES_PASSWORD=supersafe
          - POSTGRES_DB=meep_api
        volumes:
          - "postgres_data:/var/lib/postgresql/data"
```
the database it is creating in the container is called ```meep_api```. That means we are using the same database for both development and tests. Ideally we want a separate database for running tests so we can unpopulate and repopulate tables with carefully curated data in between tests. This helps keep the tests independent of one another.

I see two possible routes we can take:
  1. Change the ```TEST_DATABASE_URL``` so that the database name is something else, and then run a script in the container at startup to create the test database.
  2. Make an entirely separate block in the docker-compose file to run the tests with a database in a different container.